website/content/en/docs/reference/metrics.md

```diff
@@ -34,6 +34,30 @@ Number of nodeclaims disrupted in total by Karpenter. Labeled by reason the node
 Number of nodeclaims created in total by Karpenter. Labeled by reason the nodeclaim was created and the owning nodepool.
 - Stability Level: STABLE
 
+### `operator_nodeclaim_status_condition_transitions_total`
```

Comment:
We just need to update the deprecated metrics, not add in the new once. We should only add the new in preview and then we will move them over to v1.1 and docs after the release